packages/cli/src/commands/doctor/doctor.ts

```diff
@@ -36,22 +35,17 @@ const printIssue = ({
       : chalk.yellow('●')
     : chalk.green('✓');
 
-  const descriptionToShow = description ? `- ${description}` : '';
+  const descriptionToShow = description ? ` - ${description}` : '';
 
   logger.log(` ${symbol} ${label}${descriptionToShow}`);
 
   if (needsToBeFixed && versionRange) {
     const versionToShow = version && version !== 'Not Found' ? version : 'N/A';
-    const cleanedVersionRange = semver.valid(semver.coerce(versionRange)!);
 
-    if (cleanedVersionRange) {
-      logMessage(`- Version found: ${chalk.red(versionToShow)}`);
-      logMessage(
-        `- Minimum version required: ${chalk.green(cleanedVersionRange)}`,
-      );
+    logMessage(`- Version found: ${chalk.red(versionToShow)}`);
+    logMessage(`- Version supported: ${chalk.green(versionRange)}`);
```

Comment:
We could show the version range instead of semver coerced version. Here's a sample output how it looks now: 
![image](https://user-images.githubusercontent.com/5106466/69146170-8c811800-0acf-11ea-8bec-b6e986a541e3.png)